### PR TITLE
fix(input-stepper): add parseNumber and formatNumber to format the va…

### PR DESCRIPTION
**Pull request by gerjanvangeest**
…lue based on locale

## What I did

1.

### CODE REVIEW

File: packages/ui/components/input-stepper/test/lion-input-stepper.test.js [hunk 1]
```diff
@@ -49,6 +50,50 @@ describe('<lion-input-stepper>', () => {
     });
   });
 
+  describe('Formatter', () => {
+    it('uses formatAmount for formatting', async () => {
```
Comment (ryubro) [inline packages/ui/components/input-stepper/test/lion-input-stepper.test.js:54]: Did you mean formatNumber?

File: packages/ui/components/input-stepper/test/lion-input-stepper.test.js [hunk 2]
```diff
@@ -49,6 +50,50 @@ describe('<lion-input-stepper>', () => {
...
+      const el = await fixture(defaultInputStepper);
+      expect(el.formatter).to.equal(formatNumber);
+    });
+
+    it('formatAmount uses locale provided in formatOptions', async () => {
```
Comment (ryubro) [inline packages/ui/components/input-stepper/test/lion-input-stepper.test.js:59]: formatNumber?

File: packages/ui/components/input-stepper/test/lion-input-stepper.test.js [hunk 3]
```diff
@@ -71,7 +116,7 @@ describe('<lion-input-stepper>', () => {
       expect(el.value).to.equal('');
       el.dispatchEvent(new KeyboardEvent('keydown', { key: 'ArrowDown' }));
       await el.updateComplete;
-      expect(el.value).to.equal('-1');
+      expect(el.value).to.equal('−1');
```
Comment (ryubro) [inline packages/ui/components/input-stepper/test/lion-input-stepper.test.js:119]: What is this change about?

**Review by ryubro — APPROVED** [review]
Looks good to me.

Just a few questions / possible typo but nothing blocking.

**Review by okadurin — APPROVED** [review]

File: packages/ui/components/input-stepper/test/lion-input-stepper.test.js (hunk 1 above)
Comment (gerjanvangeest) [inline packages/ui/components/input-stepper/test/lion-input-stepper.test.js:54]:
```suggestion
    it('uses formatNumber for formatting', async () => {
```

File: packages/ui/components/input-stepper/test/lion-input-stepper.test.js [hunk 4]
```diff
@@ -49,6 +50,50 @@ describe('<lion-input-stepper>', () => {
     });
   });
 
+  describe('Formatter', () => {
+    it('uses formatNumber for formatting', async () => {
+      const el = await fixture(defaultInputStepper);
+      expect(el.formatter).to.equal(formatNumber);
+    });
+
+    it('formatAmount uses locale provided in formatOptions', async () => {
```
Comment (gerjanvangeest) [inline packages/ui/components/input-stepper/test/lion-input-stepper.test.js:59]:
```suggestion
    it('formatNumber uses locale provided in formatOptions', async () => {
```

**Pull request merged**